src/react/ssr.ts

```diff
@@ -18,7 +18,7 @@ import {deprecate} from './deprecate'
 
 const ScopeContext = React.createContext(null as Scope | null)
 export const {Provider} = ScopeContext
-function getScope() {
+export function getScope() {
   const scope = React.useContext(ScopeContext)
   if (!scope)
```

Comment:
It looks like a simplest place to introduce isomorphic behavoir to me, but maybe there a better ones 🤔 

I think, it may look like this:
```suggestion
export function getScope(forceScope?: boolean) {
  const scope = React.useContext(ScopeContext)
  if (forceScope && !scope)
```

Reply:
I think we need to move `getScope` out from the `ssr.ts` file.
Because all export from these modules is available to the end-user.